src/gfx.cpp

```diff
@@ -1185,6 +1190,44 @@ static void GfxBlitter(const Sprite * const sprite, int x, int y, BlitterMode mo
 	BlitterFactory::GetCurrentBlitter()->Draw(&bp, mode, zoom);
 }
 
+/**
+ * Draws a sprite to a new RGBA buffer (see Colour union) instead of drawing to the screen.
+ * Free the memory after use.
+ *
+ * @param spriteId The sprite to draw.
+ * @return Pixel Buffer
+ */
+const uint32* DrawSpriteToRgbaBuffer(SpriteID spriteId)
+{
+	// Gather information about the sprite to write.
```

Comment:
Code style: Use `/* */` for comments except if the comment is at the of a statement line. (More instances of this in this function).